libpebble2/services/notifications.py

```diff
@@ -42,11 +42,13 @@ def send_notification(self, subject="", message="", sender="", source=None):
         :type sender: str
         :param source: The source of the notification
         :type source: .LegacyNotification.Source
+        :param actions Actions to be sent with a notification (list of TimelineAction objects)
+        :type actions list
         """
         if self._pebble.firmware_version.major < 3:
             self._send_legacy_notification(subject, message, sender, source)
         else:
-            self._send_modern_notification(subject, message, sender, source)
+            self._send_modern_notification(subject, message, sender, source, actions)
```

Comment:
The default of `None` isn't handled anywhere; if the parameter is omitted, this is going to break.